src/components/ComponentPages/Form/ProfileInfoForm.tsx

```diff
@@ -93,13 +91,13 @@ function ProfileInfoForm({
         {suggestions.map((suggestion, index) => {
           const style = suggestion.active
             ? {
-              backgroundColor: "#f8f8f8",
-              cursor: "pointer",
-            }
+                backgroundColor: "#f8f8f8",
```

Comment:
Can you please move it to common place and use it like `colors.profile.background` ?

Reply:
Vikas has worked on this component